.github/workflows/ci.yml

```diff
@@ -273,6 +297,13 @@ jobs:
                 break
               fi
             done
+
+            # create dynamodb table
+            # aws-cli comes pre-installed in github actions
+            aws dynamodb create-table --table-name mysession --attribute-definitions AttributeName=id,AttributeType=S --key-schema AttributeName=id,KeyType=HASH --provisioned-throughput ReadCapacityUnits=1,WriteCapacityUnits=1 --endpoint-url http://localhost:8000
+            # confirm table was created
+            aws dynamodb list-tables --endpoint-url http://localhost:8000
```

Comment:
I assume this will exit with a non-zero exit code if the new "mysession" table isn't there?
Will it also echo a clear and useful error message?

Reply:
not on list-tables command (it would still work if there's no table).  If there's an exit code it would be on the create table command above, though I'm not sure if it does an exit code or not.  The list-tables command does produce a nice output in the console of the table so I think that's good enough for debugging if something goes wrong

Reply:
In that case please change the comment to reflect the fact that this is going to output the list of tables to the console, and is for debugging. The current comment makes it sound like this is doing some validation.

Reply:
Updated